examples/flutter_form_validation/pubspec.yaml

```diff
@@ -3,18 +3,14 @@ description: A new Flutter project.
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.0.0 <3.0.0"
+  sdk: ">=2.6.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
   flutter_bloc:
     path: ../../packages/flutter_bloc
-  equatable: ^0.6.0
-
-dev_dependencies:
-  flutter_test:
-    sdk: flutter
+  equatable: ^1.0.0
```

Comment:
Are you sure it is a dev dependency? You are using it in production, ergo it's a normal dependency, isn't it?

Reply:
It just removes the dependency to flutter_test.
```yaml
dependencies:
  flutter:
    sdk: flutter
  flutter_bloc:
    path: ../../packages/flutter_bloc
  equatable: ^1.0.0
```

Reply:
Oh, I see now. 😃